noxfile.py

```diff
@@ -23,10 +23,10 @@ def tests(session: Session) -> None:
         "not django",
         "-m",
         "not starlite",
-        "-m",
-        "not pydantic",
         "--ignore=tests/mypy",
         "--ignore=tests/pyright",
+        "--ignore=tests/experimental/pydantic",
+        "--ignore=tests/experimental/pydantic2",
     )
```

Comment:
otherwise pytest on the pydantic2 folder would go boom. since it tries to import stuff directly from pydantic v2.